README.md

```diff
@@ -14,7 +14,7 @@ For detection tips, see the blogpost and detection section.
 1. git clone the repo down
 2. If unsure how to create correct keys see this [blog](https://bond-o.medium.com/aws-pass-through-proxy-84f1f7fa4b4b).
 3. `pip install -r requirements.txt`
-4. [Optional] Add your slack webhook URL into config-vars.json if you want to use a webhook for when credentials are successfully guessed! (thanks to Ian for the function for notifications)
+4. [Optional] Add your slack webhook URL into config-vars.json if you want to use a webhook for when credentials are successfully guessed! 
```

Comment:
Remove this line, Notification settings will be passed via CLI (for the time being). Will be making excessive changes to the CLI input process to make this easier